web/src/lib/components/layouts/user-page-layout.svelte

```diff
@@ -30,6 +45,14 @@
           class="absolute border-b dark:border-immich-dark-gray flex justify-between place-items-center dark:text-immich-dark-fg w-full p-4 h-16"
         >
           <p class="font-medium">{title}</p>
+          {#if countTotalPeople && countTotalPeople > 0}
+            <IconButton on:click={() => (selectHidden = !selectHidden)}>
+              <div class="flex items-center">
+                <EyeOutline size="1em" />
+                <p class="ml-2">Show & hide faces</p>
+              </div>
+            </IconButton>
+          {/if}
```

Comment:
On the next line is the "button" slot. You can use that to inject a button in this spot in the html. This makes it possible to remove the above lines to another file and they will get injected here via the slot.